WordPress/gutenberg
# [RNMobile] Audio Player UI for audio block

`Gutenberg Mobile` https://github.com/wordpress-mobile/gutenberg-mobile/pull/3037
`WPAndroid` https://github.com/wordpress-mobile/WordPress-Android/pull/13843
`WPiOS` https://github.com/wordpress-mobile/WordPress-iOS/pull/15694


## Description

This PR adds a placeholder-ish Audio Player UI for the Audio Block.

## How has this been tested?




- Add an audio block.
- Check that the Audio Player UI looks as expected.

## Screenshots 


### Success
Light Mode | Dark Mode 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/105439449-e1b83c80-5c32-11eb-850f-d1ab1306e6ca.png" width="320"></kbd>  |    <kbd><img src="https://user-images.githubusercontent.com/1509205/105439447-e0870f80-5c32-11eb-996c-74af4c6c1357.png" width="320"></kbd>
   
### Uploading

Light Mode | Dark Mode 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/105439444-df55e280-5c32-11eb-839f-b777868db8cc.png" width="320"></kbd>   |    <kbd><img src="https://user-images.githubusercontent.com/1509205/105440539-b898ab80-5c34-11eb-8878-622037a2a3d1.png" width="320"></kbd>

### Error

Light Mode | Dark Mode 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/105441337-53de5080-5c36-11eb-861a-e9915a1160b9.png" width="320"></kbd>   |    <kbd><img src="https://user-images.githubusercontent.com/1509205/105441334-52ad2380-5c36-11eb-9ce2-a082fd83e253.png" width="320"></kbd>


## Checklist:
